import/popups/PopupOpener.qml

```diff
@@ -99,28 +99,28 @@ Timer {
         }
         case "embed:permissions": {
             if (data.title === "geolocation"
-                    && Popups.LocationSettings.enabled
-                    && Popups.LocationSettings.gpsPowered) {
+                    && Popups.LocationSettings.locationEnabled
+                    && Popups.LocationSettings.gpsEnabled) {
                 switch (Geolocation.response(data.host)) {
                     case "accepted": {
```

Comment:
Should we also in this case trigger a QtPositioning.startUpdates() call here?  Or does something else handle actually getting the current location and returning it to the website (e.g., is there handling for that in embedlite?)